src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -31,0 +1,94 @@
+import { Todo } from '../../types/Todo';
+import { useState, useEffect } from 'react';
+import { Option } from '../Option';
+
+interface Props {
+  allTodos?: Todo[];
+  setTodos: (value: Todo[]) => void;
+}
+
+type FilterType = 'all' | 'active' | 'completed';
+
+export const TodoFilter: React.FC<Props> = ({ allTodos, setTodos }) => {
+  const [inputSearch, setInputSearch] = useState<string>('');
+  const [close, setClose] = useState(false);
```

Comment:
The `close` state is redundant. You can determine if the input is empty directly from `inputSearch`. Consider removing `close` and using `!!inputSearch` where needed.